ql/src/test/queries/clientpositive/insert_values_orig_table_use_metadata.q

```diff
@@ -1,5 +1,6 @@
 --! qt:dataset:src
 --! qt:dataset:alltypesorc
+--! qt:replace:/(\s+totalSize\s+)\S+(\s+)/$1#Masked#$2/
```

Comment:
I don't think these lines are required now, this masking is automatically done now

Reply:
Done

Reply:
`totalSize` is not needed. We can remove it.

Reply:
Done, thanks